src/web/app/pom.xml

```diff
@@ -1625,6 +1625,38 @@
           <artifactId>gs-web-ogcapi</artifactId>
           <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+          <groupId>org.geoserver.community</groupId>
+          <artifactId>gs-web-coverages</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.geoserver.community</groupId>
+          <artifactId>gs-web-features</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.geoserver.community</groupId>
+          <artifactId>gs-web-images</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.geoserver.community</groupId>
+          <artifactId>gs-web-maps</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.geoserver.community</groupId>
+          <artifactId>gs-web-styles</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.geoserver.community</groupId>
+          <artifactId>gs-web-tiles</artifactId>
+          <version>${project.version}</version>
+        </dependency>
```

Comment:
This is probably a bad idea, considering that OGC API modules are packaged per service, e.g.:

```
> unzip -t geoserver-2.26-SNAPSHOT-ogcapi-features-plugin.zip 
Archive:  geoserver-2.26-SNAPSHOT-ogcapi-features-plugin.zip
    testing: gs-web-ogcapi-2.26-SNAPSHOT.jar   OK
    testing: snakeyaml-2.2.jar        OK
    testing: validation-api-1.1.0.Final.jar   OK
    testing: jackson-dataformat-yaml-2.17.1.jar   OK
    testing: swagger-core-2.0.2.jar   OK
    testing: swagger-models-2.0.2.jar   OK
    testing: gs-ogcapi-features-2.26-SNAPSHOT.jar   OK
    testing: gs-ogcapi-core-2.26-SNAPSHOT.jar   OK
    testing: gt-cql2-text-32-SNAPSHOT.jar   OK
    testing: jackson-dataformat-xml-2.17.1.jar   OK
    testing: GPL.html                 OK
```

This way, installing the features module alone would bring in all the descriptors for all the services. Instead, go and modify [the <service>-assembly modules](https://github.com/geoserver/geoserver/tree/main/src/community/ogcapi) to include the dependency, and eventually the assembly.xml file as well (if needed) to make sure the generated zip file has the base web module, along with the service specific one (instructions on how to use the assembly can be found in the [community README file](https://github.com/geoserver/geoserver/blob/main/src/community/README.md))

Reply:
ok - so installing `features`.zip would also install the `web-features`.
I will look into that today/tomorrow.